### PR TITLE
Added option to scale width in WidthDatacardMaker.

### PR DESCRIPTION
**Pull request by tsculac**

### CODE REVIEW

File: makeWidthDatacards.py [hunk 1]
```diff
@@ -45,6 +45,8 @@ def parseOptions():
                       help='mLow: Low m4l boundary (def=220)')
    parser.add_option('--mHigh', type='float', dest='mHigh', default="1600",
                       help='mHigh: High m4l boundary (def=1600)')
+   parser.add_option('--sw', action='store_true', dest='scale_width', default=False,
```
Comment (giacomoortona) [inline makeWidthDatacards.py]: Shouldn't the default be True? Also, I'm not sure why we should ever ask to do the integral without "width". Doesn't it come with the risk of taking the wrong normalization? (I might have missed some thread here though)

Comment (tsculac) [reply makeWidthDatacards.py]: It is just to sync on the on-shell part of Anomalous Couplings with Heshy. However I agree that the default should be set to True.

**Comment by usarica** [comment]
I don't think scale_width is needed. This datacard maker should be independent of what constant/variable binning is set. I would prefer that you only fix bugs; syncing with Heshy should be done at the level of utils/getTemplates..

**Comment by usarica** [comment]
Basically, variable binning is the primary reason why scale_width should always remain true; otherwise we cannot expect every user in general to read through this code to understand what it is doing

**Pull request closed without merging**